app/src/main/java/com/sun/note_01/screen/login/adapter/LoginViewpagerAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package com.sun.note_01.screen.login.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import com.sun.note_01.screen.login.fragment.childlogin.LoginChildFragment
+import com.sun.note_01.screen.login.fragment.childregister.RegisterChildFragment
+import com.sun.note_01.untils.Constant
+
+class LoginViewpagerAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {
+
+    private val listTab = arrayOf(Constant.TAB_LAYOUT_LOGIN, Constant.TAB_LAYOUT_REGISTER)
```

Comment:
```suggestion
    private val listTab = arrayOf(Constant.TAB_LAYOUT_LOGIN, Constant.TAB_LAYOUT_REGISTER)
```
function mới cách còn biến không cần đâu